pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -298,6 +299,38 @@ def test_iloc_setitem_bool_indexer(self, klass):
         expected = DataFrame({"flag": ["x", "y", "z"], "value": [2, 3, 4]})
         tm.assert_frame_equal(df, expected)
 
+    @pytest.mark.parametrize("dtype", ["int64", "Int64"])
+    def test_setitem_complete_columns_different_dtypes(self, dtype):
```

Comment:
this seems to work on master too.  what am i missing?

Reply:
oh woops, i was running these on my local branch that goes always-split_path.  never mind